src/components/AuthFormWrapper.tsx

```diff
@@ -17,7 +17,13 @@ export const AuthFormWrapper: React.FC<{ children: React.ReactNode }> = (
         justify='center'
       >
         {/* <Heading color='blue.900'>[OpenCampus]</Heading> */}
-        <Box w='60' bg='gray.200' h='10' />
+        {/* <Box w='60' bg='gray.200' h='10' /> */}
+        <Image
+          htmlHeight='80px'
+          htmlWidth='300px'
+          objectFit='contain'
+          src='../../public/images/opencampus-logo.png'
```

Comment:
This path should not be relative.
The path should be `/images/opencampus-logo.png`